### PR TITLE
Pymarl branch

### PR DESCRIPTION
**Pull request by PaLeroy**
This PR will be accepted once:

- Everything can be executed properly from the root.
- A script to retrieve seed of any algo_env config.
- A step by step guide to reproduce experiments has been created.

### CODE REVIEW

**Comment by PaLeroy** [comment]
TODO:
- merge the seed scripts
- Execute the whole instructions to ensure no remaining errors.

File: REPROD_EXP.md
Comment (moratodpg) [inline REPROD_EXP.md:1]: This is really great! 💯

File: imp_env/pomdp_models/generate_transitions.ipynb
Comment (moratodpg) [inline imp_env/pomdp_models/generate_transitions.ipynb:1]: We already have updated this in the relevant branch (imp_env). Shall we keep this commit here as well?

**Comment by moratodpg** [comment]
I added a couple of minor comments. I understand we have to further test this before merging.

**Review by moratodpg — APPROVED** [review]

File: imp_env/pomdp_models/generate_transitions.ipynb
Comment (PaLeroy) [reply imp_env/pomdp_models/generate_transitions.ipynb:1]: It will be updated in accordance I believe :)

**Review by PaLeroy — COMMENTED** [review]
I believe everything is fine.
Everything worked for me.

**Pull request merged**